frontend/src/pages/about/About.tsx

```diff
@@ -0,0 +1,310 @@
+            <Typography paragraph>
+              We are also supported by the software company PolyConseil, in the
+              context of their #Tech4Good program. .... accompanying our
+              software development through organizational support. Since August
+              2021, PolyConseil is also allocating an engineering intern one day
+              per week to developing Tournesol and is proposing UX/UI designs.
+              We are very grateful for their help.
+            </Typography>
+          </ContentBox>
+        </Grid>
+
+        <Grid item xs={12} md={6} className={classes.container}>
+          <ContentBox className={classes.card}>
+            <Typography variant="h4">Kleis</Typography>
+            {/* TODO: logo */}
+            <Typography paragraph>
+              , as well as by the software company Kleis. through organizational
+              support, ... and more
+            </Typography>
+          </ContentBox>
+        </Grid>
+      </Grid>
+
+      <Grid container className={classes.root}>
+        <Grid item xs={12} md={6} className={classes.container}>
+          <ContentBox>
+            <img height="64px" src="/logos/Foss_Logo.png" />
+            <Typography variant="h4">Open Source Contributions</Typography>
+            <Typography paragraph>
+              As Tournesol is an open source project, we have been lucky to
+              benefit from contributions by multiple volunteers. Find our
+              wonderful contributors on{' '}
+              <a
+                href="https://github.com/tournesol-app/tournesol/graphs/contributors"
+                target="_blank"
+                rel="noreferrer"
+              >
+                Github Contributors
+              </a>
+            </Typography>
+          </ContentBox>
+        </Grid>
+      </Grid>
+
+      <Grid
+        container
+        className={classes.root}
+        style={{ background: '#1282B2', color: 'white' }}
+      >
+        <Grid item xs={12} className={classes.container}>
+          <ContentBox>
+            <Typography variant="h1">Public Database</Typography>
+            <Typography paragraph>
+              Tournesol aims to benefit research ... Our public database can be
+              downloaded by clicking here (TODO). It is published under{' '}
+              <a
+                href="https://opendatacommons.org/licenses/by/1-0/"
+                style={{ color: 'white' }}
+              >
+                Open Data Commons Attribution License (ODC-By)
+              </a>
+              .
+            </Typography>
+            <Typography paragraph>
+              Possible thanks to our users ... Finally, we would like to thank
+              all of the volunteers who compared videos on Tournesol. We count
+              so far about 2500 users who compared 9000 times more than 3000
+              videos.
+            </Typography>
+          </ContentBox>
+        </Grid>
+      </Grid>
```

Comment:
Would it make sense to explain this in a dedicated page, where the content of the dataset could be described in detail?

Reply:
Yes, I was quite confused about the content of this "About" page, which seems to be mostly "Who contributes to Tournesol" page. Let's wait a bit until I get a final version, then discuss it when we have a more concrete thing to look at.